package.json

```diff
@@ -13,6 +13,7 @@
     "react-native": "0.63.4",
     "react-native-calendars": "^1.1247.0",
     "react-native-easy-grid": "0.2.2",
+    "react-native-gesture-handler": "^1.10.0",
```

Comment:
this shall also be not needed? `    "react-native-gesture-handler": "^1.10.0",`

Reply:
removed